image/overlays/windows10/autounattend.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
These xmls does not need drivers for storage e.g. like in https://github.com/kubevirt/ssp-operator/blob/master/data/tekton-pipelines/windows-efi-installer-configmaps.yaml#L20?

Reply:
Since virtio is not used for storage they are not required.